wiki/Guides/How_To_Get_Your_Map_Ranked/en.md

```diff
@@ -1,25 +1,22 @@
 
-Your map's star priority is important when it comes to moving on to the next step. If a modder particularly likes your map they might give it a kudosu star - the more you work to make your map better, the more likely this will be. You can also gain stars by modding other people's maps and spending the kudos on your own mapset. If you don't think you're very good at modding, practice makes perfect! Even a few small suggestions can really help another mapper out, so don't be afraid to give modding a go.
+If a modder likes your map, they might give it hype. Hype allows your mod to get noticed more by Beatmap Nominators. Your map's hype train is important when it comes to moving on to the next step.
 
-## Step 2: Calling a BN
+## Step 2: Calling a Beatmap Nominator
 
-Once your map has been pending a while, has a relatively high star priority, and you think it's perfect and ready to be ranked, it could be time to call up a [Beatmap Nominator](/wiki/People/Beatmap_Nomination_Group). Make sure you check the team member in question's signature or userpage to see whether they have a modding policy: if they're open to requests or there's nothing there, ask them politely in game if they wouldn't mind taking a look at your map. Lots of BN are extremely busy — there are an awful lot of pending maps to get through, so don't be too disheartened if you are turned down. Having plenty of mods and kudos stars on your map will help encourage BN to look at your map, so if you're having difficulties throwing a few more kudos stars your map's way might improve your chances. Once a BN has modded your map and you've made the fixes necessary, call them back to take another look and see whether it is worthy of a bubble.
+Once your map has been pending for a while, has 5 people on your hype train, and you think it's perfect and ready to be ranked, it's time to get a [Beatmap Nominator](/wiki/People/Beatmap_Nomination_Group). Make sure you check the team member's signature or userpage to see whether they have a modding policy; if they're open to requests or there's nothing there, ask them politely in game if they wouldn't mind taking a look at your map. Lots of BN are extremely busy — there are an awful lot of pending maps to get through, so don't be too disheartened if you are turned down. Having plenty of mods and hype on your map will help encourage Beatmap Nominators to look at your map, so if you're having difficulties getting a Beatmap Nominator to look at your map, remember that your hype train and mods will help you out.
 
 ## Step 3: Bubbles
```

Comment:
seems to be more about what to do after nominations, not about nominations directly